src/components/Timeline/TimelineNote.tsx

```diff
@@ -73,31 +70,50 @@ const NoteMessage: React.FC<NoteMessageProps> = ({ message }) => (
   </>
 );
 
+const TimelineAvatar = ({
```

Comment:
question: don't you thingk about move this component to a separate file?

Reply:
This file has multiple components. `TimelineAvatar` is not intended to be used anywhere else apart from `TimelineNote` so I think it can stay here "privately" 😄